Assets/Scripts/Globals.cs

```diff
@@ -41,15 +41,16 @@ public static Vector3 DirectTargeting(Transform from, Transform to)
     {
         { EnemyType.TURRET, "Turret" },
         { EnemyType.HEALER, "Healer" },
```

Comment:
This too

Reply:
we will need something like this for variants and their prefabs